_data/meltano/extractors/tap-launchdarkly/pulumi.yml

```diff
@@ -0,0 +1,55 @@
+capabilities:
+- about
+- catalog
+- discover
+- schema-flattening
+- state
+- stream-maps
+description: Obtain Feature Flags from LaunchDarkly
+domain_url: https://launchdarkly.com/
+executable: tap-launchdarkly
+keywords:
+- meltano_sdk
+label: Launchdarkly
+logo_url: /assets/logos/extractors/launchdarkly.png
+maintenance_status: active
+name: tap-launchdarkly
+namespace: tap_launchdarkly
+next_steps: ''
+pip_url: git+https://github.com/pulumi/tap-launchdarkly.git
+quality: bronze
+repo: https://github.com/pulumi/tap-launchdarkly
+settings:
+- description: The token to authenticate against the API service
+  kind: password
+  label: Auth Token
+  name: auth_token
+- description: Environment to get activation data for
+  kind: string
+  label: Environment
+  name: environment
+  value: production
```

Comment:
Is this value always available?

Reply:
It's a mandatory setting, the tap extracts one environment at a time

Reply:
Do you mean if it's possible to use LaunchDarkly without an environment? All the entities sit in an environment if that's the question
![image](https://github.com/meltano/hub/assets/4952116/1c042b04-908f-459e-bd65-c32e8845a2f2)

Reply:
Gotcha. My worry is that it's an arbitrary value. For example, a user may have named their production environment `prod` instead. Does that make sense? If so, we should probably remove the default value and instead use [`settings_group_validation`](https://docs.meltano.com/reference/plugin-definition-syntax/#settings_group_validation).

Reply:
Ohhhh yes you are right

Reply:
Addressed in https://github.com/meltano/hub/pull/1595/commits/4197f1ecfae6293f2bb92b336811321082b99ba7